.github/workflows/bundle.yaml

```diff
@@ -0,0 +1,35 @@
+name: bundle
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  bundle:
+    name: bundle
```

Comment:
nit: I believe the name of this job would be `bundle/bundle` in the actions UI -- do we want to name it something more descriptive? Maybe `bundle/deploy` is appropriate?